### PR TITLE
Add FuGet link

### PR DESCRIPTION
**Pull request by Krzysztof-Cieslak**
FuGet is a NuGet package browser combined with an API browser. It's a really useful tool for .Net developers and it would be good to expose it directly from the package page. It enables developers to inspect content of NuGet packages, the API of packages, and even API diffs between different versions of the package. Here's example page - https://www.fuget.org/packages/FSharp.Core 

The PR adds link to FuGet page for the current package in the Info section of the page.

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -9,6 +9,15 @@
 
     var absolutePackageUrl = Url.Absolute(Url.Package(Model.Id));
 
+    var fugetPackageUrl = "";
+    var search = "nuget";
+    var replace = "fuget";
+    int pos = absolutePackageUrl.IndexOf(search);
+    if (pos >= 0)
```
Comment (agr) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: I have a feeling, this code will not properly work on dev and int enviornments: https://dev.nugettest.org and https://int.nugettest.org.

**Comment by Krzysztof-Cieslak** [comment]
PR updated.

**Comment by bgrainger** [comment]
FuGet is great, but I have concerns about its long-term stability/viability. https://www.fuget.org/ is currently down (Error 524 from CloudFlare) and the repo itself (https://github.com/praeclarum/FuGetGallery) hasn't had any activity on issues or PRs for 3-4 months.

Unless there are strong reasons to believe that it will be available and supported for the foreseeable future, it seems potentially rash to add a link to it to the NuGet Gallery.

**Comment by anangaur** [comment]
I like the functionality fuget.org has and I feel exposing it to more developers will benefit them. However, the current proposal needs some changes:
* The UX needs to indicate it will open up a new webpage, probably in a new tab?
* It needs to have a disclaimer text indicating the link points to a third party website, not owned by Microsoft.
* Comply with accessibility requirements (in case we have tooltip, icons etc.)

I have created an issue to track the progress of these changes: https://github.com/NuGet/NuGetGallery/issues/7850. 

In addition, the concerns raised by @bgrainger on the reliability and maintainability of the fuget.org service are valid. Looping in @praeclarum (fuget.org owner) to get his views on the concerns raised above and PR in general.

**Comment by chgill-MSFT** [comment]
Plans as of now for the items @anangaur mentioned:

**Icon (this is for show, svg will be used):** Image

**Link tooltip:** “Explore additional package info on fuget.org”
**Info (i) icon tooltip:** “fuget.org is a 3rd party website, not controlled by Microsoft. This link is made available to you per the NuGet Terms of Use.”

**Link accessibility:** "Link, open in fuget.org explorer"
**Info (i) icon accessibility:** “fuget.org is a 3rd party website, not controlled by Microsoft. This link is made available to you per the NuGet Terms of Use.”

**Comment by agr** [comment]
The engineering requirements for the change are pretty much outlined in my initial comment:
* The display of the link should be opt-in and disabled by default as having it enabled only makes sense for nuget.org alone;
  * It can be done by using feature flags (through `IFeatureFlagService` already available in `PackagesController`);
  * Alternatively, a configuration option can be added to do the same;
* The link generation should be tested;
  * The code that generates the link should go to the [`PackagesController`](https://github.com/NuGet/NuGetGallery/blob/84ea5890b77e79b1f218521f2c031f8ee6c311f5/src/NuGetGallery/Controllers/PackagesController.cs#L808) and be passed through the model, this will put the business logic that generates it to an appropriate location and make testing it easier, tests then would go to [`PackagesControllerFacts`](https://github.com/NuGet/NuGetGallery/blob/84ea5890b77e79b1f218521f2c031f8ee6c311f5/tests/NuGetGallery.Facts/Controllers/PackagesControllerFacts.cs#L396).

@Krzysztof-Cieslak, would you be willing to update the code to accommodate that?

**Comment by chgill-MSFT** [comment]
Here's the SVG file for the icon: [fuget.org icon](https://gist.github.com/chgill-MSFT/397c8aae7aaf16feb711d4f03bef2993#file-fuget-icon-svg)

**Comment by chgill-MSFT** [comment]
@Krzysztof-Cieslak have you had a chance to take a look the suggested additions for the icon, tooltip, and screen reader accessibility? I believe @agr also outlined some engineering requirements 😊

**Comment by drewgillies** [comment]
FYI: as a suitable amount of time as passed and we'd like to see this feature in-product, I've gone ahead and coded up this one as a replacement: https://github.com/NuGet/NuGetGallery/pull/8364

**Comment by drewgillies** [comment]
Now that the new change is merged, I'll go ahead and close this. Thanks for spurring us to action, @Krzysztof-Cieslak!

**Pull request closed without merging**